src/api/requests/submitUpdateFinancialProfile.ts

```diff
@@ -92,7 +92,7 @@ const submitUpdateFinancialProfile = async (
 ): Promise<null> => {
   return request<URLSearchParams, null>({
     axiosInstance: mailApiClient,
-    url: `/send`,
+    url: `/public/case/send`,
```

Comment:
Super appreciate you updating this to keep things working and updated with the backend. Jason updated the base of this url in the configmap on dev, so we'd end up with `{mail-api-url}/public/case/public/case/` I think. I'll double check with him.

Reply:
Yeah, we can drop this change since it should just be a change in the `.env`/ configmap.

Reply:
Maybe just update the values here:
https://github.com/search?q=repo%3Acfpb%2Fsbl-frontend+SBL_MAIL_BASE_URL%3D&type=code

Then we'll get this PR approved and shipped.